core/src/main/java/com/homeaway/streamplatform/streamregistry/resource/BaseResource.java

```diff
@@ -25,12 +25,13 @@
 public class BaseResource {
 
     protected Response buildErrorMessage(Response.Status status, Exception e) {
-        log.error(e.getMessage(), e);
+        ErrorMessage errorMessage = new ErrorMessage(status.getStatusCode(),
+                e.getMessage(),
+                e.getCause() != null ? e.getCause().getMessage() : null);
 
+        log.error(errorMessage.toString(), e);
         return Response.status(status)
-                .entity(new ErrorMessage(status.getStatusCode(),
-                        e.getMessage(),
-                        e.getCause() != null ? e.getCause().getMessage() : null))
+                .entity(errorMessage)
                 .build();
     }
 
```

Comment:
Earlier, the log message did not print Response Code.

Reply:
Sounds great. Do you think unit test is necessary so that we don't get a regression ?